webapp/src/containers/genePage/index.js

```diff
@@ -94,9 +94,11 @@ class GenePage extends Component {
           <OrthologyTable data={mockOrthologData} />
         </Subsection>
 
+        {this.props.data.diseases.length ?
         <Subsection hardcoded title='Disease Associations'>
-          <DiseaseTable />
-        </Subsection>
+          <DiseaseTable data={this.props.data}/>
```

Comment:
Can this just pass the `disease` field of `data` instead of the whole thing? Might be nice to limit the scope of the data that each component works with to reduce dependencies.